blt/src/Commands/HumsciConfigCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Acquia\Blt\Custom\Commands;
+
+use Acquia\Blt\Robo\Commands\Setup\ConfigCommand;
+use Acquia\Blt\Robo\Exceptions\BltException;
+
+/**
+ * Class HumsciCommands.
+ *
+ * @package Acquia\Blt\Custom\Commands
+ */
+class HumsciConfigCommand extends ConfigCommand {
+
+  /**
+   * Imports configuration from the config directory according to cm.strategy.
+   *
+   * @command drupal:config:import
+   * @aliases dci setup:config-import
+   *
+   * @option partial Don't delete existing configuration.
+   *
+   * @validateDrushConfig
+   * @executeInVm
+   */
+  public function import($options = ['partial' => FALSE]) {
+    $strategy = $this->getConfigValue('cm.strategy');
+    $cm_core_key = $this->getConfigValue('cm.core.key');
+    $this->logConfig($this->getConfigValue('cm'), 'cm');
+
+    if ($strategy != 'none') {
+      $this->invokeHook('pre-config-import');
+
+      $task = $this->taskDrush()
+        ->stopOnFail()
+        // Sometimes drush forgets where to find its aliases.
+        ->drush("cc")->arg('drush')
+        // Rebuild caches in case service definitions have changed.
+        // @see https://www.drupal.org/node/2826466
+        ->drush("cache-rebuild")
+        // Execute db updates.
+        // This must happen before features are imported or configuration is
+        // imported. For instance, if you add a dependency on a new extension to
+        // an existing configuration file, you must enable that extension via an
+        // update hook before attempting to import the configuration.
+        // If a db update relies on updated configuration, you should import the
+        // necessary configuration file(s) as part of the db update.
+        ->drush("updb");
+
+      // If exported site UUID does not match site active site UUID, set active
+      // to equal exported.
+      // @see https://www.drupal.org/project/drupal/issues/1613424
+      $exported_site_uuid = $this->getExportedSiteUuid($cm_core_key);
+      if ($exported_site_uuid) {
+        $task->drush("config:set system.site uuid $exported_site_uuid");
+      }
+
+      switch ($strategy) {
+        case 'core-only':
+          $this->importCoreOnly($task, $cm_core_key);
+          break;
+
+        case 'config-split':
+          try {
+            $this->importConfigSplit($task, $cm_core_key, $options['partial']);
```

Comment:
All this to get `partial` in here?

Reply:
yeah because i can't override the importConfigSplit() method without first taking over the `drupal:config:import` command.

Reply:
Yeah, Bummer.